### ISSUE TITLE
How to use csvimport.py

### ISSUE THREAD

**Issue by jimmyzhangau** [MISP/misp-modules#273]
Hi,
I googled and dug a lot, but still couldn't find any instruction on how to use this module. Can anyone point me to any link on how to use this module? Currently, I know I have to edit userConfig to reflect the header of my csv, can anyone post a example here? After I edited userConfig, do I run the module using 'python3 csvimport.py mycsvfile.csv' ? Do I need to configure any authkey for this module to import events into MISP? If no, what authentication method to be used for the module talking to MISP instance?

Cheers,
Jimmy

**Comment by saeeda12** [comment]
Hi @jimmyzhangau - did you ever figure out how to use this module? Currently confused myself.

**Comment by jimmyzhangau** [comment]
Hi Amal,
No luck for me. So finally I used a STIX to MISP script developed by a independent developer to import STIX format intelligence. Keep me posted if you are lucky and find something. :)

Cheers,
Jimmy

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by saeeda12** [comment]
Hi Jimmy,

I actually got the CSV import working!

I found this PDF: https://www.circl.lu/assets/files/misp-training/brussels2016/misp-modules.pdf

But mainly I just followed this part: `sudo vi /etc/rc.local`, and added this line: 'sudo -u www-data misp-modules -s &'` while in the `/usr/local/src/misp-modules` directory. I didn't see this in the installation script.

Then I made sure my url in the Plugin settings was set to `http://127.0.0.1` and port to '6666' - I was initially trying to use a different port, but it wasn't working, and tried https before but it threw an error with SSL.

Then I entered `curl -s http://127.0.0.1:6666/modules` from my MISP box, and it returned all the available import modules - I also saw this in the Plugin settings when I dropped down the Import section. From there, I enabled the **Plugin.Import_csvimport_enabled** by setting the value to true, and when I went to an event and clicked on "Populate from", the CSVimport option was available.

I actually tracked the issues I was having in this issue #4401. Hope this helps you out!
